vignettes/blueprint/intro.Rmd

```diff
@@ -8,27 +8,27 @@ vignette: >
   %\VignetteEncoding{UTF-8}
 ---
 
-`teal` is an interactive data exploration framework based on Shiny, designed with emphasis on analyzing CDISC clinical trial data.
+`teal` is an interactive data exploration framework based on Shiny, designed primarily to analyze CDISC clinical trial data.
 
 A Shiny Application created with `teal` offers users the ability to:
 
 - Import data from external sources;
 - Dynamically filter data for analysis;
-- Generate reproducible code for future analysis
-- Create downloadable reports of analysis results _(for analysis modules that support reporting)_.
+- Generate reproducible code for future analysis;
+- Create and download reports of analysis results _(for analysis modules that support reporting)_.
 
-Moreover, the `teal` framework provides application developers with a wide range of customizable standard analysis modules to integrate into their applications, along with a logging framework that helps facilitate debugging.
+Moreover, the `teal` framework provides application developers with a wide range of customizable standard analysis modules to integrate into their applications, along with a logging framework that facilitates debugging.
 Additionally, advanced users of the framework can develop new analysis modules and easily integrate them into any `teal` application.
 
 The `teal` framework's functionality draws heavily from the following packages:  
 
 | R package            | Description                                                             |
```

Comment:
I know you haven't changed this but I spotted this and it might be an easy enhancement

Should we redirect to the doc page instead of GitHub? E.g. https://insightsengineering.github.io/teal/ instead of https://github.com/insightsengineering/teal. IMHO GitHub should not be user-facing (except reporting issues)